components/developer/jenkins/LICENSE

```diff
@@ -0,0 +1,23 @@
+MIT License 
```

Comment:
Reasonable point. Copied from some other recipe, one of those this was stitched from :) Probably that one pulled a file from GNU-style sources.